config.cfg

```diff
@@ -1,14 +1,13 @@
 [user]
-marketLocale = FR
+marketLocal = GB
```

Comment:
This is a breaking change. The code specifically looks for `marketLocale`. Additionally, there is no need to change the country from FR to GB in the repo. Make all config changes locally.